test/e2e-go/features/transactions/lease_test.go

```diff
@@ -235,17 +235,9 @@ func TestOverlappingLeases(t *testing.T) {
 	_, err = client.BroadcastTransaction(stx2)
 	a.Error(err)
 
-	// wait for us to be building leaseStart + 10, the last round where
-	// the first txn's lease is still valid
-	fixture.WaitForRoundWithTimeout(leaseStart + 9)
-
-	// submitting the second transaction should still fail
-	_, err = client.BroadcastTransaction(stx2)
-	a.Error(err)
-
-	// wait for us to be building leaseStart + 11, where the first txn's
+	// wait for us to be building leaseStart + 21, where the first txn's
 	// lease should have expired
-	fixture.WaitForRoundWithTimeout(leaseStart + 10)
+	fixture.WaitForRoundWithTimeout(leaseStart + 20)
```

Comment:
The value '20' here is chosen based on the '20' set earlier. 
It will be nicer to have a variable to hold this value, and use is in both location. 
Importantly, the comments and the name of the variable can be descriptive enough to understand what and why is the value 20 for.

Reply:
made this a const